README.md

```diff
@@ -0,0 +1,58 @@
+# BIOL:4386 Course Project - Reproducing *Deriving genotypes from RAD-seq short-read data using Stacks*
+## Project Homework 1 - Feb 12 2023
+
+## Reference 
+Rochette, N., Catchen, J. Deriving genotypes from RAD-seq short-read data using Stacks. Nat Protoc 12, 2640–2659 (2017). https://doi.org/10.1038/nprot.2017.123
+
+## Introduction 
+Use of restriction-site associated DNA sequencing (RAD seq) has increasingly been utilized to genotype populations without non-model organisms as a tool for population genetics analysis. To produce a reliable set of loci for downstream analysis, understanding of bioinformatic programs such as STACKS is necessary. 
```

Comment:
I assume you eventually want to apply this to your project. If so, add a paragraph explaining the overall goal and background of your project. It's fine, I should say, to construct this course project solely on the basis of learning the tool. adding some biological question and background, however, would make it more relatable. you can, for example, give an intro to your project as a way of explaining the advantage RAD sequencing brings and the unique challenges that come with it. this can help you better motivate what you want to reproduce in the project.